documentation/common/attributes-kubernetes.adoc

```diff
@@ -0,0 +1,36 @@
+
+
+// :namespace: controls the CLI command
+:cmdcli: kubectl
+// :namespace: controls the word used for namespace in
+:namespace: namespace
+
+// Name and version placeholders
+:ProductLongName: Strimzi
+:ProductName: Strimzi
+:ContextProduct: strimzi
+:OpenShiftName: OpenShift
+:OpenShiftLongName: OpenShift Origin
+:OpenShiftVersion: 3.9 and later
+:KubernetesVersion: 1.9 and later
+:KubernetesName: Kubernetes
+:KubernetesLongName: Kubernetes
+:ProductPlatformName: {KubernetesName}
+:ProductPlatformLongName: {KubernetesLongName}
+:AProductPlatformName: A {KubernetesName}
```

Comment:
The attributes that include the article, we don't use these for new content?  We just add the article outside the attribute? We will have sentences where the definite article is used too, I guess, so our aim will be to revert back to using {ProductPlatformName} only?

Reply:
I don't understand what you're proposing. Writing "A {ProductPlatformName}" is problematic because `{ProductPlatformName}` can be "OpenShift", which breaks the rules of English grammar. And there are places where we need to use a lower case "a", which necessitates `_aProductPlatformName` (because asciidoc attributes are case insensitive). Of course, if you can construct sentences without the indefinite article then you can just use `ProductPlatformName`. And if you did that everywhere in the doc we could get rid of these attributes.

If you have a better way of doing it then I'm really happy to consider it. @scholzj suggested pre-processing the `.adoc` using `sed`, which would make authoring simpler, but is ugly in different ways. I just don't see any nice way of doing this.

Reply:
Thanks Tom. It wasn't about the approach so much as what was expected going forward, whether this was to fix the current content and we took a different approach to new content. I can see the advantages of using the articles within the variables and it won't be hard to adopt this new strategy, but I do wonder if an initial search and replace would have done a lot of the hard work.